themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -129,7 +129,6 @@
                 type="number"
                 value="{$product.quantity}"
                 name="product-quantity-spin"
-                min="{$product.minimal_quantity}"
```

Comment:
You should keep this one maybe? :thinking:

Reply:
This html 5 min attribute was useful because it automatically fixed the value according to minimal quantity at page load. 

So for example if a user had one item of a product in his cart, and in BO the minimal quantity for this product is changed to 10, then when the user refreshes his cart, the html 5 forces the value the minimal quantity.

But as seen with @colinegin we don't want this behavior anymore, we want to keep the old value (even if it's below the minimal quantity required), and it's the user's job to change the value before continuing his order. So that he doesn't order more products that he intented to do in the first place. Instead we display an error message (and he can't click on 'order' because of the error message).

I checked that this template is only used for the cart.

Reply:
So be careful with https://github.com/PrestaShop/PrestaShop/blob/develop/themes/_core/js/cart.js#L94 and https://github.com/PrestaShop/PrestaShop/blob/develop/themes/_core/js/product.js#L156
and others :roll_eyes:

Reply:
You're right, I'll check it out 🤔

Reply:
@PierreRambaud 

So I checked, and in those two files we are dealing with the 'min' attribute from the quantity input in the **product** page. I only removed the one on the **cart** page.

I was going to add a custom data attribute, but even in the cart.js, the min attribute that is used comes from the product page (the value is used when we click on "add to cart")

I couldn't find any use of the 'min' attribute for the quantity field in the **cart** page.

Reply:
Good news :+1: